internal/confmapprovider/discovery/bundle/bundle.d/receivers/smartagent-collectd-kafka-broker.discovery.yaml

```diff
@@ -0,0 +1,47 @@
+#####################################################################################
+#                               Do not edit manually!                               #
+# All changes must be made to associated .tmpl file before running 'make bundle.d'. #
+#####################################################################################
+smartagent/collectd/kafka_broker:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)kafka.*broker"}) and not (command matches "splunk.discovery")
```

Comment:
I think the broker/consumer/producer distinction is likely going to be detected by reported metrics based on available mbeans and not running process identifiers. Having the same rules for all kafkas makes more sense to me out of the gate.